### PR TITLE
Degrade performance for licensing reasons

### PR DESCRIPTION
**Pull request by dirkgr**
This is a knock-on effect of the effort to open-source science-parse.

### CODE REVIEW

**Comment by siegeln** [comment]
LGTM. @hsuhanooi we should keep this in mind if we ever need to rebuild pdffigures2 for figure extraction.

File: README.md [hunk 1]
```diff
@@ -36,9 +36,16 @@ To install it you will need an appropriate resolver like:
 
 Then include
 
-`libraryDependencies += "org.allenai" %%  "pdffigures2" % "0.0.6"`
+`libraryDependencies += "org.allenai" %%  "pdffigures2" % "0.0.10"`
 
-Including this package in [jcenter](https://bintray.com/bintray/jcenter) is in progress.
+For licensing reasons, PDFFigures2 does not include libraries for some image formats. If you have no
+licensing restrictions in your project, we recommend you add these additional dependencies to your
```
Comment (schmmd) [inline README.md]: Can you add a sentence on why people would add the following dependencies?

Comment (dirkgr) [inline README.md]: Done

**Comment by dirkgr** [comment]
I suspect I have to find the figure extraction job and add in these extra dependencies there.

**Comment by dirkgr** [comment]
Turns out FE in the pipeline doesn't use this project. So it's all good.

**Pull request merged**